js/main.js

```diff
@@ -0,0 +1,45 @@
+//Happy Birthday Leader! 🎇💙
+
+const card_list = document.getElementById("card-list");
+const CLOUD_NAME = "dazcxdgiy";
+const CLOUDINARY_URL = `https://res.cloudinary.com/${CLOUD_NAME}/image/upload/`;
```

Comment:
I'm surprised there's no API secret stuff here. I think that makes sense since we're only "reading" from Cloudinary. API keys (and keeping that secret) only come into play if we're concerned about making upload/ post calls. So there's no security risk of our cloud name being public. Is that right?

Reply:
Correct, the cloud name is public and can be seen on all image urls, so it's safe to show client side. The API keys are only used at build time to upload the images, so they're set as environment variables inside Netlify.